### DISCUSSION TITLE
crash of widelands

### DISCUSSION THREAD

**Discussion by BlackMage2** [widelands/widelands#6475] [General]
Hello, i have everytime a crash of widelands when i try to start it. i have compiled it from source

[2024-06-29T17.59.36_139942885689344.txt](https://github.com/user-attachments/files/16041348/2024-06-29T17.59.36_139942885689344.txt)

**Comment by BlackMage2** [comment]
what I had make false?

**Reply by BlackMage2** [reply]
must i create the config file manual?

**Reply by LucasTheCure** [reply]
Hi, not sure but in the log you have `/lib/x86_64-linux-gnu/libc.so.6` which is related to your OS. What system are you using? I installed dependencies then run ./compile and game loaded without issues on Ubuntu 22.04

**Comment by tothxa** [comment]
First of all, please use issues for problem reporting. Then you even get a template to guide you what information to provide to help solving your problem.

In this case (beyond the obvious what system you are using and what steps you did), using the `--verbose --verbose-i18n` command line options may or may not help identify better where and why the problem occurs. Maybe a better backtrace from a debug build (with ASan) could also give some more clues. The raw backtrace already shows that the crash happens during early startup (in `WLApplication::handle_commandline_parameters()`, in a call to `translate()` (a.k.a. `_()`).  Most likely you hit a previously untested code path (we've had a major overhaul of the translation code recently), where `translate()` is called before the default textdomain is set up, violating the assertion in line 169 of `base/i18n.cc`... oops, that means you don't actually get a backtrace in a debug build, only an assertion failure... if that happens, please delete that line and recompile again.

To answer your other question: No, the config file is created automatically.
